mapie/regression/regression.py

```diff
@@ -315,7 +315,7 @@ def _check_agg_function(
                 "You need to specify an aggregation function when "
                 f"cv's type is in {self.cv_need_agg_function_}."
             )
-        elif (agg_function is not None) or (self.cv in self.no_agg_cv_):
+        elif agg_function is not None:
```

Comment:
I'm removing this restriction to make the code simpler (unnecessary if condition) because the `_aggregate_with_mask` in `estimator.py` does the job and the `_check_agg_function` returns a default value in all cases.